### ISSUE TITLE
Cannot multi-arch build a container 

### ISSUE THREAD

**Issue by cdrage** [containers/podman-desktop#6236]
### Bug description

Go to build any Containerfile and select both AMD64 & ARM64.

You'll get this output when trying to build:

```
Uploading the build context from /Users/cdrage/syncthing/dev/electron/bootc/examples/bootc-helloworld-python-systemd...Can take a while...
Error:(HTTP code 400) unexpected - failed to parse query parameter 'platform': "linux/arm64,linux/amd64": invalid platform syntax for --platform="linux/arm64,linux/amd64": "arm64,linux" is an invalid componError:Error: (HTTP code 400) unexpected - failed to parse query parameter 'platform': "linux/arm64,linux/amd64": invalid platform syntax for --platform="linux/arm64,linux/amd64": "arm64,linux" is an invalid component of "linux/arm64,linux/amd64": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument 
```

![Screenshot 2024-03-01 at 1 53 26 PM](https://github.com/containers/podman-desktop/assets/6422176/83ce6045-7ed8-445b-8085-61fb9f036f00)

### Operating system

mac M1

### Installation Method

None

### Version

1.7.0

### Steps to reproduce

1. Build any Containerfile.
2. Select both arm64 and amd64
3. Will not build / syntax error.

**Comment by afbjorklund** [comment]
This duplicates from the command-line too, if you use `docker` instead of `podman-remote`:

```console
$ podman-remote build --platform "linux/arm64,linux/amd64" testbuild
[linux/amd64] STEP 1/2: FROM busybox
[linux/amd64] STEP 2/2: RUN true
--> Using cache f163439e98770295b7256eb910969a40c7d8290e006f8f4d40019f018cc93f33
--> f163439e9877
[linux/arm64] STEP 1/2: FROM busybox
f163439e98770295b7256eb910969a40c7d8290e006f8f4d40019f018cc93f33
Resolved "busybox" as an alias (/home/anders/.cache/containers/short-name-aliases.conf)
Trying to pull docker.io/library/busybox:latest...
Getting image source signatures
Copying blob sha256:c2bf9493c1bf786e95e3eac7d406c20aa1b8a2d40916756e891627e9e8f8d119
Copying config sha256:3e4fd538a9a0b729be05707cf805388be2fb701cfd5d44c6542f1988e8aef6e3
Writing manifest to image destination
[linux/arm64] STEP 2/2: RUN true
--> Using cache 7378c5c81721fa1ad4ee97335dca29f91ac881451f345f09f177f297d0234e38
--> 7378c5c81721
7378c5c81721fa1ad4ee97335dca29f91ac881451f345f09f177f297d0234e38
$ docker build --platform "linux/arm64,linux/amd64" testbuild
[+] Building 0.0s (0/0)                                                                                                                                                          
ERROR: multiple platforms feature is currently not supported for docker driver. Please switch to a different driver (eg. "docker buildx create --use")
$ DOCKER_BUILDKIT=0 docker build --platform "linux/arm64,linux/amd64" testbuild
DEPRECATED: The legacy builder is deprecated and will be removed in a future release.
            BuildKit is currently disabled; enable it by removing the DOCKER_BUILDKIT=0
            environment-variable.

Sending build context to Docker daemon  2.048kB
Error response from daemon: "arm64,linux" is an invalid component of "linux/arm64,linux/amd64": platform specifier component must match "^[A-Za-z0-9_-]+$": invalid argument
```

**Comment by cdrage** [comment]
~Investigation so far is that this may be an upstream issue, I have opened up: https://github.com/containers/podman/issues/22071~

TLDR:

Not an upstream issue. We'll have to implement manifests / support them to do the equivalent of:
```
# First, initialise the manifest
podman manifest create <image name>

# Build the image attaching them to the manifest
podman build --platform linux/amd64,linux/arm64  --manifest <image name>  .

# Finally publish the manifest
podman manifest push <image name>
```

but in podman desktop

**Comment by vrothberg** [comment]
Came up in https://github.com/redhat-developer/podman-desktop-redhat-account-ext/issues/112.

**Comment by cdrage** [comment]
With https://github.com/containers/podman-desktop/pull/6792 merged, this can now be closed. I have opened up https://github.com/containers/podman-desktop/issues/6815 to further enhance the fixed feature post-summit.
